DOD-ISD-Verbs/dodisdverbs.jsonld

```diff
@@ -0,0 +1,4729 @@
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disposed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disposed"
+			},
+			"definition": {
+				"en": "Eliminating something or someone by throwing away, removing, or destroying."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/disseminated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "disseminated"
+			},
+			"definition": {
+				"en": "Spread or disperse something, especially information widely."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/drove",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "drove"
+			},
+			"definition": {
+				"en": "Propel or carry along by force in a specified direction; operate and control the direction and speed of a vehicle."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/egressed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "egressed"
+			},
+			"definition": {
+				"en": "Go out of or leave a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/elevated",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "elevated"
+			},
+			"definition": {
+				"en": "Raise or lift something up to a higher position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/emplaced",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "emplaced"
+			},
+			"definition": {
+				"en": "Put in place or position."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/employed",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "employed"
+			},
+			"definition": {
+				"en": "Make use of something or someone; provide work to someone for payment."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/engaged",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "engaged"
+			},
+			"definition": {
+				"en": "Occupy, attract, or involve someone; enter into conflict or combat with an enemy."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/energized",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "energized"
+			},
+			"definition": {
+				"en": "Give energy, vitality and/or enthusiasm to someone or something."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/entered",
+			"inScheme": "https://w3id.org/xapi/dod-isd",
+			"@type": "Verb",
+			"prefLabel": {
+				"en": "entered"
+			},
+			"definition": {
+				"en": "Come or go into a place."
+			}
+		},
+		{
+			"@id": "https://w3id.org/xapi/dod-isd/verbs/established",
```

Comment:
This seems underspecified. Also, it seems strange that the value can only be one of Knowledge, Skills, or Attitude. Is it not possible for multiple to apply to a statement? (I can't be sure due to the lack of description)

Reply:
The verbs are specific to DoD use and are classified such that a verb only covers one type of learning - knowledge, skill, or attitude. We could add that data to the verb ahead of time instead of making it a variable.